pkg/templates/aws/resources/ecr_image.yaml

```diff
@@ -16,6 +16,7 @@ properties:
         direction: downstream
         resources:
           - aws:ecr_repo
+        unique: true
```

Comment:
If the edge is `many-to-one`, is this unique flag still correct?

Reply:
yeah we would have to do some refactoring to make this work otherwise. This is just saying we prefer 1:1 but we wont prevent many:1